### PR TITLE
The Big Public API Documentation update, volume 2.

### PR DESCRIPTION
**Pull request by stewemetal**
Another milestone on the road to the full docs update.

Still only inline public API doc and guide updates.

### CODE REVIEW

**Comment by stewemetal** [comment]
@lejard-h @JEuler Another (now smaller! 😉 ) docs update PR is up for review. This is still not the whole thing, but I'm getting there.

**Comment by stewemetal** [comment]
@lejard-h @JEuler No rush guys, but could you please take a look at this in a couple of days? 😉

**Comment by JEuler** [comment]
Yeah, of course, sorry, was moving inside the country. Now I'm home, will have a look soon!

File: chopper/README.md [hunk 1]
```diff
@@ -2,21 +2,35 @@
...
-[Documentations](https://hadrien-lejard.gitbook.io/chopper)
+[**Documentation**](https://hadrien-lejard.gitbook.io/chopper)
 
-## Installation
+## Adding Chopper to your project
+
+In your project's `pubspec.yaml` file, 
+
+* Add *chopper*'s latest version to your *dependencies*.
+* Add `build_runner: ^1.10.3` to your *dev_dependencies*.
```
Comment (JEuler) [inline chopper/README.md:12]: Should we upgrade the build_runner? :)

**Review by JEuler — APPROVED** [review]

File: chopper/README.md (hunk 1 above)
Comment (stewemetal) [reply chopper/README.md:12]: Could you please test the lib with the latest one?

Also, bumping the `build_runner` version should be a different PR IMHO. 😄 

@JEuler @lejard-h Please note, that I've created a [milestone](https://github.com/lejard-h/chopper/milestone/1) for the next, 3.0.7 release, so please link any further PRs to it on the right until the next release to keep track of the changes easier! 😉 

I'd like to do proper changelogs starting with 3.0.7.

Comment (JEuler) [reply chopper/README.md:12]: Oops, missed that. Of course, this should be a different PR, just asking :) I'm using the fresh build runner in the production app with Chopper - seems okay. :)

**Pull request merged**